src/main/java/com/kodilla/ecommercee/controller/OrderController.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.GenericEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/order")
+public class OrderController {
+
+    @RequestMapping(method = RequestMethod.GET, value = "getOrders")
+    public List<GenericEntity> getOrders() {
+        return new ArrayList<>();
+    }
+
+    @RequestMapping(method = RequestMethod.POST, value = "createOrder")
+    public void createOrder() {
+    }
+
+    @RequestMapping(method = RequestMethod.GET, value = "getOrder")
+    public GenericEntity getOrder(Long id) {
```

Comment:
Tutaj też bym dodał  `@PathVariable`. Tylko że z tego co widzę w poprzednich modułach było robione tak jak Ty napisałeś więc może żeby nie wporwadzać zamętu zostaw tak jak jest, a w `ProductController` się to poprawi